server/src/entity/Review.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  BaseEntity,
+  Column,
+  Entity,
+  CreateDateColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+import { Account } from './Account';
+import { Restaurant } from './Restaurant';
+import { MenuItem } from './MenuItem';
+
+@Entity()
+export class Review extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  public id: string;
+
+  @CreateDateColumn()
+  public dateReviewed: Date;
+
+  @Column({
+    default: 0,
+  })
+  public restaurantReviewScore: number;
+
+  @Column({
+    default: '',
+  })
+  public restaurantReviewComment: string;
+
+  @Column({
+    default: 0,
```

Comment:
why allow empty review score?

Reply:
the person can review either item or restaurant. review is a general term

Reply:
according to this schema, in theory, we could have a completely empty review. You may want to make either two separate review entities. Or, just make it a generic review entity and allow it to have a foreign key to either a restaurant or a menu item so that we can tell which review it is for

Reply:
okay so we keep both manytoone and keep only the reviewComments and reviewScore?